src/app/modules/sgid/sgid.errors.ts

```diff
@@ -0,0 +1,43 @@
+import { ApplicationError } from '../../modules/core/core.errors'
+
+export class CreateRedirectUrlError extends ApplicationError {
```

Comment:
suggest prefixing names with `Sgid` or similar for ease of interpretation against other errors in the logs